components/x-eventpromo/src/Details.jsx

```diff
@@ -0,0 +1,23 @@
+import { h } from '@financial-times/x-engine';
+import Footer from './Footer';
+import Link from './Link';
+import Meta from './Meta';
+
+const Details = ({link, title, ...props}) => {
+
+	return (
+		<div className="event-promo-inarticle__blocks">
```

Comment:
I think this needs a heading element (even if hidden and referenced using an `aria-labelledby` attribute) to identify this content